src/main/java/online/store/book/mapper/BookMapper.java

```diff
@@ -12,6 +13,8 @@
 public interface BookMapper {
     BookDto toDto(Book book);
 
+    List<BookDto> fromEntityListToDtoList(List<Book> books);
```

Comment:
```suggestion
    List<BookDto> toDto(List<Book> books);
```
let's just use method overloading